lib/react.dart

```diff
@@ -563,16 +614,14 @@ abstract class Component2 implements Component {
   /// See: <https://facebook.github.io/react/docs/react-component.html#updating-componentwillreceiveprops>
```

Comment:
Can we remove the note in the comment above about `componentWillReceivePropsWithContext`?